ci/azure-pipelines-nightly.yml

```diff
@@ -48,6 +48,25 @@ jobs:
     artifact: opentitan-repo
     displayName: "Upload repository"
 
+- job: hyperdebug
+  displayName: "Hyperdebug tests"
+  timeoutInMinutes: 30
+  dependsOn: checkout
+  pool: FPGA
+  steps:
+  - template: ../ci/checkout-template.yml
+  - template: ../ci/install-package-dependencies.yml
+  - bash: |
+      set -x
+      module load "xilinx/vivado/$(VIVADO_VERSION)"
```

Comment:
I can see the other jobs using this, so I'm sure it correct, but how does `"xilinx/vivado/$(VIVADO_VERSION)"` work?

Doesn't `$(...)` run commands and `${...}` expand variables? I see `VIVADO_VERSION` is a `variable:` above, but does Azure convert that to a command?

Reply:
This is an [azure syntax](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#understand-variable-syntax): `${{ }}` gets expanded by Azure at compile-time of the YAML, while `$()` gets expanded at runtime by Azure before the task runs. Admitedly it's a bit confusing because it's also a bash expression.

Reply:
Ah okay, thank you